### DISCUSSION TITLE
doris multivariate data directory usage problem

### DISCUSSION THREAD

**Discussion by yanghuisr** [apache/doris#16332] [A - General / Q&A]
doris version：1.2.1.1
hive version：1.2.1
The hive catalog is created according to the official documents:
![1675245803536](https://user-images.githubusercontent.com/124145208/216012305-a2b7c51a-3ea3-4fd1-aae1-d6ed33cbd699.png)

The hive version is specified during creation：
![1675245929336](https://user-images.githubusercontent.com/124145208/216012749-1958e076-b9b8-41a0-a3e1-0973ea812d10.png)

After I created the hive catalog, I successfully executed statements such as "show databases" and "show tabsles"

It was also successful the first time I used the select statement to query a table, but the following problem occurred when I switched to query other tables:
![1675246458380](https://user-images.githubusercontent.com/124145208/216014766-95949c2a-17af-4177-b113-67e077f00175.png)

The query statement format is as follows:
![1675246248459](https://user-images.githubusercontent.com/124145208/216014030-46fbc087-69d1-4973-94cb-433c227bd72f.png)

When I tried to "refresh" the built catalog, the error occurred even with the previous statement "show databases"
![1675246458380](https://user-images.githubusercontent.com/124145208/216014766-95949c2a-17af-4177-b113-67e077f00175.png)

I feel very confused about this and hope to get help from the community leaders

**Comment by LemonLiTree** [comment]
Which version of the hive jar package in the fe/lib directory?

**Reply by yanghuisr** [reply]
fe/lib hive.version:2.3.7
![1675308298411](https://user-images.githubusercontent.com/124145208/216224010-225495b2-4be0-4ac8-acd8-f63291f2f463.png)

**Comment by LemonLiTree** [comment]
https://github.com/apache/doris/pull/15886 
Hive 1. x is supported in 1.2.2，It will be released soon

**Reply by yanghuisr** [reply]
Ok, thank you very much！
